raiden_libs/messages/monitor_request.py

```diff
@@ -22,95 +20,70 @@ class MonitorRequest(Message):
 
     def __init__(
         self,
-        channel_identifier: ChannelIdentifier,        # uint256
-        nonce: int,                     # uint64
-        transferred_amount: int,        # uint256
-        locksroot: str = None,               # str -> bytes 32
-        extra_hash: bytes = None,         # bytes 32
-        signature: bytes = None,          # bytes
+        balance_proof: BalanceProof,
+        non_closing_signature: str = '',
```

Comment:
Should this be `None` by default?

Reply:
yes none would be better, I'll update & rebase